### ISSUE TITLE
Store train and val loss as an optional metric in registry

### ISSUE THREAD

**Issue by ab93** [numaproj/numalogic#79]
# Summary
Storing training loss and validation loss would provide more insights in the model look up.

---

**Message from the maintainers**:

If you wish to see this enhancement implemented please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.
